src/gtp_context.erl

```diff
@@ -745,9 +745,9 @@ get_handler_if(Socket, #gtp{version = v2} = Msg) ->
     gtp_v2_c:get_handler(Socket, Msg).
 
 find_sender_teid(#gtp{version = v1} = Msg) ->
-    gtp_v1_c:find_sender_teid(Msg);
+    gtp_v1_c:find_sender_teid(gtp_packet:decode_ies(Msg));
```

Comment:
This will lead to decoding the IEs twice. The IE decoding is delayed by intention. It is expensive and should not happen in the socket process.
Most errors that require a TEID in the response will happen in places where the IEs are already decoded. Change those places to return the sender TEID in their error response.